src/components/Wallet/WalletDeposit.jsx

```diff
@@ -50,47 +53,41 @@ class WalletDeposit extends React.Component {
     clearInterval(depositTimer);
   }
 
-  getDepositableNodes = () => {
```

Comment:
This was happening on every change to the current address, and also caused tracking of the next address to be more complicated. Using selectors should be more efficient and avoids complications with state tracking.